todo_app/app.py

```diff
@@ -8,7 +9,13 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return render_template('index.html', items=get_items())
+
+
+@app.route('/', methods=['POST'])
+def addNewItem():
```

Comment:
Method names in Python should be snake case (so `add_new_item` rather than `AddNewItem`)

Reply:
Ah ok, I was taught to use camelCase when I learnt Java. Will use snake case from now on 👍